src/components/Button/Button.tsx

```diff
@@ -103,6 +107,11 @@ export const Button = chakra(
           __css={styles}
           {...rest}
         >
+          {screenreaderOnlyText && (
+            <Box as="span" __css={styles.screenreaderOnly}>
+              {screenreaderOnlyText}
+            </Box>
+          )}
```

Comment:
The screen reader only text should be placed after the standard button content.  In fact, it would be best if it was immediatelt folloowing the text and before an icon.  However, I am not sure that is possible with the way we currently compose the button text and icon.

Reply:
I'll move it after the entire `children`. It'd be more work to insert this element between the text and the icon, but possible. The real issue, I think, is that the order is not always "text -> icon". A dev can pass in "icon -> text" so inserting in between those elements will not always provide the ideal placement.

Reply:
Yes, understood.  We'll go with these latest updates.